source/configuration_manual/authentication/ldap_conf_ext_settings.rst

```diff
@@ -0,0 +1,603 @@
+.. _authentication-ldap_conf_ext_settings:
+
+============================
+Settings in ldap conf ext
+============================
+
+This page enlists settings being configured in the ldap configuration files
+(e.g, conf.d/auth-ldap.conf.ext) for userdb, passdb or retieving sieve script
+from ldap.
+
+.. Note:: The ldap configuration files are opened as root, so should be owned by root and mode 0600.
+
+
+.. _ldap_conf_ext_setting-auth_bind:
+
+``auth_bind``
+---------------
+
+- Default: ``no``
+- Values:  :ref:`boolean`
+
+Set yes to use authentication binding for verifying password's validity.
+This works by logging into LDAP server using the username and password given by client.
+The :ref:`ldap_conf_ext_setting-pass_filter` is used to find the DN for
+the user. Note that the pass_attrs is still used, only the password field
+s ignored in it. Before doing any search, the binding is switched back

```

Comment:
"s" -> "is"

Reply:
corrected. however because my local repository got corrupted and I had to commit only the latest updated file, log for this fix is missing.